src/scripts/i18n/de.json

```diff
@@ -0,0 +1,227 @@
+{
+    "allArticles": "Alle Artikel",
+    "add": "Hinzufügen",
+    "create": "Erstellen",
+    "icon": "Icon",
+    "name": "Name",
+    "openExternal": "Extern öffnen",
+    "emptyName": "Dieses Feld darf nicht leer sein.",
+    "emptyField": "Dieses Feld darf nicht leer sein.",
+    "edit": "Berarbeiten",
+    "delete": "Entfernen",
+    "followSystem": "Folge System",
+    "more": "Mehr",
+    "close": "Schließen",
+    "search": "Suchen",
+    "loadMore": "Lade mehr",
+    "dangerButton": "Bestätige {action}?",
+    "confirmMarkAll": "Möchtest Du wirklich alle Artikel als gelesen markieren?",
+    "confirm": "Bestätigen",
+    "cancel": "Abbrechen",
+    "time": {
+        "now": "gerade eben",
+        "m": " Minuten",
+        "h": "h",
+        "d": " Tage",
```

Comment:
Is there any specific reason for the acronym of minute and day to be written in full?

Reply:
German has not really a abbreviation for Minutes and Days.

Reply:
I'll leave this as-is for now. Hope the length of the time won't interfere with the UI too much.